src/index.js

```diff
@@ -411,11 +428,16 @@ class CreditCardInput extends Component<Props, State> {
     };
   };
 
-  setFieldInvalid = (errorText: string) => {
+  setFieldInvalid = (errorText: string, inputName) => {
     const { invalidClassName } = this.props;
     // $FlowFixMe
     document.getElementById('field-wrapper').classList.add(invalidClassName);
     this.setState({ errorText });
+
+    if (inputName) {
+      const { onError } = this.props[`${inputName}InputProps`];
+      onError && onError(errorText);
```

Comment:
Can we actually make the argument of `onError` an object with keys `inputName` and `error` perhaps? The consumer may want the name of the input.